howto/serialize/README.md

```diff
@@ -0,0 +1,27 @@
+# Serialization in Dapr's SDKs
+
+An SDK for Dapr should provide serialization for two use cases. First, for transient objects sent through request and response payloads. Second, for objects to be persisted. For both these use cases, a default serialization must be provided. In the Java SDK, it is the [DefaultObjectSerializer](https://dapr.github.io/java-sdk/io/dapr/serializer/DefaultObjectSerializer.html) class.

```

Comment:
Lining up these 2 cases with the set of user-exposed apis that use them will make this more concrete.

Reply:
Done.